### PR TITLE
Fix: clear orientation listener on close

### PR DESCRIPTION
**Pull request by ldalzottomp**
## What

This PR cleans android orientation listener on camera session close.
If this is not done, the events are still fired and they accumulate until you kill the app.

## Changes

Added a `clearListeners` functions in the OrientationManager which is then called when CameraSession.close() is called.

## Tested on

Samsung Galaxy tab active4 pro. Android 14.

### CODE REVIEW

**Review by Kamiss7 — APPROVED** [review]

File: package/android/src/main/java/com/mrousavy/camera/core/OrientationManager.kt [hunk 1]
```diff
@@ -67,13 +67,17 @@ class OrientationManager(private val context: Context, private val callback: Cal
     }
   }
 
+  // remove previous listeners if attached
```
Comment (mrousavy) [inline package/android/src/main/java/com/mrousavy/camera/core/OrientationManager.kt]: Can you move this comment down again to line 80 where it was before?

File: package/android/src/main/java/com/mrousavy/camera/core/OrientationManager.kt [hunk 2]
```diff
@@ -67,13 +67,17 @@ class OrientationManager(private val context: Context, private val callback: Cal
...
+  fun clearListeners() {
```
Comment (mrousavy) [inline package/android/src/main/java/com/mrousavy/camera/core/OrientationManager.kt]: Let's rename this to `stopUpdates`

**Review by mrousavy — CHANGES_REQUESTED** [review]
Hey - thanks for your PR! This is good, just a few code style nitpicks.

**Comment by ldalzottomp** [comment]
Thanks for handling the requested changes

**Pull request merged**